packages/a11y/src/add-intro-text.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Build the explanatory text to be placed before the aria live regions.
```

Comment:
Just to clarify: There's nothing here which explicitly associates the intro element to the other live regions, but we create them as sibling elements in an order which expect to be sufficient for how screen readers would navigate between the elements?

Reply:
Yes. It just relies on the order.